src/systems/triggered_publisher/TriggeredPublisher.cc

```diff
@@ -596,33 +598,81 @@ void TriggeredPublisher::Configure(const Entity &,
     }
   }
   else
-  {
     ignerr << "No ouptut specified" << std::endl;
-    return;
+
+  if (sdfClone->HasElement("service"))
+  {
+    for (auto serviceElem = sdfClone->GetElement("service"); serviceElem;
+         serviceElem = serviceElem->GetNextElement("service"))
+    {
+       ServiceOutputInfo s_info;
```

Comment:
we are using CammelCase

```suggestion
       ServiceOutputInfo serviceInfo;
```

Reply:
Ah good to know! Thanks! I'll change accordingly. Since I don't have much experience in code review, I appreciate all your feedback (: Keep them coming! +1: